### PR TITLE
OSOE-91: Normalize line endings to auto across all submodules

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-91](https://lombiq.atlassian.net/browse/OSOE-91)

### CODE REVIEW

File: wwwroot/.placeholder [hunk 1]
```diff
@@ -1 +0,0 @@
-��
```
Comment (0liver) [inline wwwroot/.placeholder:1]: Why did you delete this file?

**Review by 0liver — CHANGES_REQUESTED** [review]

File: wwwroot/.placeholder (hunk 1 above)
Comment (sarahelsaig) [reply wwwroot/.placeholder:1]: As long as we push the contents of the _wwwroot_ directory a placeholder is unnecessary. Also the directory was recreated by Gulp even if you deleted it, so it was double pointless. Also its presence might be confusing since the directory is not empty.

**Review by 0liver — APPROVED** [review]

**Review by 0liver — APPROVED** [review]
